aiosmtpd/controller.py

```diff
@@ -50,9 +52,11 @@ def __init__(
         port=8025,
         *,
         ready_timeout=1.0,
-        enable_SMTPUTF8=None,
         ssl_context: ssl.SSLContext = None,
-        server_kwargs: Dict[str, Any] = None
+        # SMTP parameters
+        server_hostname: str = None,
```

Comment:
Need a kwargs that does NOT conflict with `Controller`'s `hostname` kwarg.

In 2.0 we'll drop `hostname` from `Controller`, because the _semantically proper_ name should be `host` not `hostname`.